docs/content/build.md

```diff
@@ -26,6 +26,9 @@ To build the worker configs, you can either:
 
 > **It’s finished building when:** there is a message stating `'spatial build build-config' succeeded`.
 
+<%(Callout type="alert" message="If you encounter build errors, you might not have selected Linux Build Support during your setup. You need Linux build support because server-workers in a cloud deployment run in a Linux environment. In the `Assets/Fps/Config/BuildConfiguration`, do not change the `UnityGameLogic Cloud Environment` from Linux.")%>
```

Comment:
You only get the Linux Build Support option if you install Unity using the Installer. This is often the cause of people not clicking Linux Build Support (because they never had the option, because they chose the Executable rather than the Installer). Perhaps this should be called out explicitly?